guides/doc-Upgrading_and_Updating/topics/post_upgrade_tasks.adoc

```diff
@@ -3,3 +3,14 @@
 Some of the procedures in this section are optional. You can choose to perform only those procedures that are relevant to your installation.
 
 If you use the PXE-based discovery process, then you must complete the discovery upgrade procedure on {Project} and on any {SmartProxyServer} with hosts that you want to be listed in {Project} on the *Hosts* > *Discovered hosts* page.
+
+* The `katello-agent` method is disabled with a new installation of {Project} {ProjectVersion}, making both the `qpidd` and `qdroutered` services unavailable.
+
+* If a {Project} {ProjectVersionPrevious} is upgraded to a {Project} {ProjectVersion}, the `katello-agent` method, as well as the `qpidd` and `qdroutered` services, remain enabled.
+
+* If you are not using `katello-agent` and transitioned to remote execution, you can optionally disable the `katello-agent` method properly as a post-upgrade task for both {Project} {ProjectVersion} and {SmartProxy} {ProjectVersion}:
```

Comment:
What does properly mean?

Reply:
"Properly" isn't necessary here. I will remove this and method.